### ISSUE TITLE
Via Licensing

### ISSUE THREAD

**Issue by kennytv** [ViaVersion/Via-Meta#1]
We have decided to license ViaVersion under the GNU GPLv3 in order to be in line with Bukkit API being licensed under it, as well as in an effort to promote OSS over publicly distributed proprietary software - excluding an extra API module.
For formality's sake, we ask our contributors to give us permission to re-license their contributed code as such, this includes contributions to https://github.com/ViaVersion/ViaVersion, https://github.com/ViaVersion/ViaBackwards, https://github.com/ViaVersion/ViaRewind, https://github.com/ViaVersion/ViaFabric, and https://github.com/ViaVersion/ViaRewind-Legacy-Support.

We are aware that this does not affect any of the previous versions of our projects and that any code looked at by viewing previous versions/commits is still licensed under the permissive MIT license and can be used accordingly.

**Comment by Gerrygames** [comment]
I give you permission to re-license my contributed code under the GNU GPL

**Comment by Foorcee** [comment]
I give you permission to re-license my contributed code under the GNU GPL

**Comment by creeper123123321** [comment]
I give you permission to re-license my contributed code under the GNU GPL

**Comment by FormallyMyles** [comment]
I give you permission to re-license my contributed code under the GNU GPL

**Comment by RaphiMC** [comment]
I give you permission to re-license my contributed code under the GNU GPL

**Comment by astei** [comment]
Although you technically already have permission to do so, I grant explicit permission to re-license my contributions to ViaVersion under the GNU GPL.

**Comment by Xernium** [comment]
I give you permission to re-license my contributed code under the GNU GPL

**Comment by tomcraft** [comment]
I give you permission to re-license my contributed code under the GNU GPL

**Comment by MrMicky-FR** [comment]
I give you permission to re-license my contributed code under the GNU GPL

**Comment by Pokechu22** [comment]
I give you permission to re-license my contributed code under the GNU GPL

**Comment by sgdc3** [comment]
I give you permission to re-license my contributions to Via projects under the GNU GPL

**Comment by vectrixdevelops** [comment]
I give you permission to re-license my contributed code under the GNU GPL

**Comment by Camotoy** [comment]
I give you permission to re-license my contributed code under the GNU GPL

**Comment by Jo0001** [comment]
I give you permission to re-license my contributed code under the GNU GPL

**Comment by kashike** [comment]
👍

**Comment by Logics4** [comment]
I give you permission to re-license my contributed code under the GNU GPL

**Comment by Redned235** [comment]
I give you permission to re-license my contributions to Via projects under the GNU GPL.

**Comment by Earthcomputer** [comment]
I give you permission to re-license my contributed code under the GNU GPL

**Comment by Plancke** [comment]
I give you permission to re-license my contributed code under the GNU GPL

**Comment by 7kasper** [comment]
Yes, you have my permission for relicensing under GNU GPL. Good luck!

**Comment by Paulomart** [comment]
I give you permission to re-license my contributed code under the GNU GPL

**Comment by hugokerstens** [comment]
I give you permission to re-license my contributed code under the GNU GPL

**Comment by lenis0012** [comment]
I give you permission to re-license my contributions to Via and related projects under the GNU GPL license.
Thanks for reaching out.

**Comment by retrooper** [comment]
I give you my permission to re-license my denied contributions to ViaVersion under the GNU GPL3 license.

**Comment by Matsv** [comment]
I give you permission to re-license my contributed code under the GNU GPL (:
